deploy/Chart/templates/database/configmaps.yaml

```diff
@@ -1,13 +1,23 @@
 apiVersion: v1
-kind: ConfigMap
+kind: Secret
 metadata:
   name: database-secret
+  namespace: {{ .Release.Namespace }}
+  labels:
+    control-plane: database
+    app.kubernetes.io/part-of: radius
+stringData:
+  POSTGRES_USER: "{{ .Values.database.postgres_user }}"
+  POSTGRES_PASSWORD: "{{ randAlphaNum 16 }}"
```

Comment:
Just an observation: If we want to connect to this DB using a DB editor for debugging purposes let's say, we are going to need to get this password.